.github/workflows/publish-major-minor.py

```diff
@@ -0,0 +1,17 @@
+import sys
+
+from packaging.version import parse
+
+print(sys.version_info)
+tag_ref = sys.argv[1]
+tag_name = tag_ref.split("/")[-1]
+print(f"tag_name: {tag_name}")
+version = parse(tag_name)
+print(f"version: {version}")
+if not (version.is_prerelease):
```

Comment:
It's quite untypical to use a C-notation in Python conditionals, let's drop it.
```suggestion
if not version.is_prerelease:
```

Reply:
Ups that is an artifact from when used `is_prerelease` and `is_devrelease`, before I realized that `is_devrelease` for rc releases aswell.